### PR TITLE
small fixes and additions

### PR DESCRIPTION
**Pull request by PrimePlaya24**
- Fixed archive backup method from backing up itself if archive was in specified backup folder
  - For example, attempting to backup "./BT_backup/" and storing the archive under "./BT_backup/archive.zip"
  - Caused recursive loop until space runs out
- Better string formatting for logging
  - Supplied message as parameter rather than doing string interpolation beforehand
  - https://stackoverflow.com/a/5082809/8612448
- strip() and lower() when checking input
- Use more f-strings

### CODE REVIEW

**Comment by jslay88** [comment]
#18 closes this

File: qbt_migrate/classes.py [hunk 1]
```diff
@@ -86,8 +90,9 @@ def discover_relevant_fast_resume(cls, bt_backup_path: str, existing_path: str,
     @classmethod
     def backup_folder(cls, folder_path: str, archive_path: str):
         cls.logger.info(f'Creating Archive {archive_path} ...')
+        files_to_archive = os.listdir(folder_path)
```
Comment (PrimePlaya24) [inline qbt_migrate/classes.py:93]: This fix was not implemented in #18 merge. This stops the issue of archiving the archive file if the backup dir was set to the archive path.

**Comment by jslay88** [comment]
Sorry, waited forever for requested changes to be made and they weren’t.
Was dealing with 7 issues at once. Will deal with it later.

**Comment by PrimePlaya24** [comment]
No problem! Thanks for your hard work.

**Comment by jslay88** [comment]
#21

**Pull request closed without merging**